js/form.js

```diff
@@ -0,0 +1,33 @@
+const adForm = document.querySelector('.ad-form');
+const mapForm = document.querySelector('.map__filters');
+
+const arrAdForm = [...adForm];
+const arrMapForm = [...mapForm];
+
+const getBlockForms = () => {
+  adForm.classList.add('ad-form--disabled');
+  mapForm.classList.add('ad-form--disabled');
+
+  arrAdForm.forEach((element) => {
+    element.setAttribute('disabled', 'disabled');
```

Comment:
```suggestion
    element.disabled = true;
```
Отдавай предпочтение такому способу